### PR TITLE
Fix: deprecated strftime with IntlDateFormatter

### LINKED ISSUE #34: [K4] 4.0 Compatibility
> Function strftime() is deprecated
> \site\plugins\kirby-calendar-plugin-master\snippets\calendar-table.php:28

Kirby 4.0.0-rc.4
PHP 8.1.23

### PR DESCRIPTION
**Pull request by rmeister**
A little band-aid for the deprecation of strftime.

### CODE REVIEW

**Comment by mzur** [comment]
Thanks! Is this intentionally still a draft?

File: snippets/calendar-div.php [hunk 1]
```diff
@@ -1,6 +1,9 @@
 <?php
 	$tmpDate = getdate(0);
 	$currentDate = getdate();
+  if (!empty($languageCode)) {
```
Comment (mzur) [inline snippets/calendar-div.php:4]: This should probably be:
```suggestion
  if (!isset($languageCode)) {
```

right?

File: snippets/calendar-table.php [hunk 2]
```diff
@@ -1,6 +1,9 @@
 <?php
 	$tmpDate = getdate(0);
 	$currentDate = getdate();
+  if (!empty($languageCode)) {
```
Comment (mzur) [inline snippets/calendar-table.php:4]:
```suggestion
  if (!isset($languageCode)) {
```

File: snippets/calendar-teaser.php [hunk 3]
```diff
@@ -1,12 +1,15 @@
 <ul class="teaser">
 <?php
+  if (!empty($languageCode)) {
```
Comment (mzur) [inline snippets/calendar-teaser.php:3]:
```suggestion
  if (!isset($languageCode)) {
```

**Review by mzur — COMMENTED** [review]
Thanks a lot! Do you use this already on a site somewhere (and everything works)?

I noticed that some changes replaced the old tab indentation with spaces. Could you please change the indentation back to tabs?

**Comment by rmeister** [comment]
I use it on https://hog-neppendorf.de
It seems there still is an issue that could be related to timezones, as begin dates are incorrectly printed as one day before the specified day.

**Comment by mzur** [comment]
@rmeister Does everything work for you now?

**Comment by rmeister** [comment]
Yes, looks good so far :)

**Review by mzur — APPROVED** [review]
Thanks!

**Pull request merged**